skillserver/echo.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 // Request Functions
+// Deprecated: Please use the github.com/mikeflynn/go-alexa/customskill package
 func (this *EchoRequest) VerifyTimestamp() bool {
```

Comment:
I haven't yet implemented this in `customskill` so perhaps the deprecation notice shouldn't exist here. I would be tempted to add this as an optional property of `Skill` which specified the maximum age a request would be valid for. Thoughts?